.database_consistency.yml

```diff
@@ -7,3 +7,50 @@ ActiveStorage::Blob:
   enabled: false
 ActiveStorage::VariantRecord:
   enabled: false
+
```

Comment:
This is like a Rubocop Ignore rule list.
Added comments explaining why we're ignoring these rule breaks.